packages/salesforcedx-vscode-core/src/util/configUtil.ts

```diff
@@ -75,4 +80,10 @@ export class ConfigUtil {
     const apiVersion = await ConfigUtil.getConfigValue('apiVersion');
     return apiVersion ? String(apiVersion) : undefined;
   }
+
+  public static async getAllAliasesFor(username: string): Promise<string[]> {
+    const stateAggregator = await StateAggregator.getInstance();
+    const aliases = stateAggregator.aliases.getAll(username);
+    return aliases;
+  }
```

Comment:
OK to use StateAggregator here in configUtil, or would we prefer to have a separate StateUtil module as a wrapper around StateAggregator?

Reply:
I think it's fine here.  We probably want to have some kind of CoreService at some point, but this isn't the time.